src/main/scala/org/constellation/ConstellationExecutionContext.scala

```diff
@@ -66,4 +69,10 @@ object ConstellationExecutionContext {
       new DefaultThreadFactory("callbacks")
     )
   )
+
+  def createSemaphore[F[_]: Concurrent](): Semaphore[F] = {
```

Comment:
Can we pass there a number of available permits?
`createSemaphore(n: Long)`

Reply:
Fixed